### PR TITLE
 LTD-3697-make-recommendation-ncsc 

### PR DESCRIPTION
**Pull request by depsiatwal**
This fixes make recommendation doesn't appear for NCSC users. 

Also depends on database update to have the associated alias which will follow

### CODE REVIEW

File: unit_tests/caseworker/advice/test_services.py [hunk 1]
```diff
@@ -134,7 +138,6 @@ def test_get_advice_for_countersign_with_post_circ_countersigning(current_user,
     (True, "final", False, LICENSING_UNIT_TEAM, LU_SR_LICENSING_MANAGER_QUEUE, "cases:advice_view", {"review_and_countersign": False},),
     # An individual accessing the case after giving countersigned advice
     (True, "user", True, BEIS_NUCLEAR, BEIS_NUCLEAR_COUNTERSIGNING, "cases:countersign_view", {"edit_recommendation": True, "move_case_forward": True},),
-    (True, "user", True, FCDO_TEAM, FCDO_COUNTERSIGNING_QUEUE, "cases:countersign_view", {"edit_recommendation": True, "move_case_forward": True},),
```
Comment (saruniitr) [inline unit_tests/caseworker/advice/test_services.py:137]: Is there a reason to remove this?

Comment (depsiatwal) [reply unit_tests/caseworker/advice/test_services.py]: ohh reverted and also added another test

File: caseworker/advice/services.py [hunk 2]
```diff
@@ -45,6 +46,7 @@
     "MOD_CAPPROT",
 ]
 LU_CONSOLIDATE_TEAMS = [FCDO_TEAM, MOD_ECJU_TEAM]
+NCSC_TEAM = "NCSC"
```
[the remark this answers is not in the document]
Comment (depsiatwal) [reply caseworker/advice/services.py:49]: Nope but as mentioned in comments PR will follow to update DB

File: unit_tests/caseworker/advice/test_services.py [hunk 3]
```diff
@@ -116,6 +118,8 @@ def test_get_advice_for_countersign_with_post_circ_countersigning(current_user,
     (False, "user", False, MOD_CONSOLIDATE_TEAMS[1], MOD_CONSOLIDATE_QUEUES[2], "cases:advice_view", {"make_recommendation": True},),
     (False, "user", False, MOD_CONSOLIDATE_TEAMS[2], MOD_CONSOLIDATE_QUEUES[3], "cases:advice_view", {"make_recommendation": True},),
     (False, "user", False, MOD_CONSOLIDATE_TEAMS[3], MOD_CONSOLIDATE_QUEUES[4], "cases:advice_view", {"make_recommendation": True},),
+    (False, "user", False, MOD_CONSOLIDATE_TEAMS[3], MOD_CONSOLIDATE_QUEUES[4], "cases:advice_view", {"make_recommendation": True},),
```
Comment (saruniitr) [inline unit_tests/caseworker/advice/test_services.py:121]: this line is a duplicate of line above this one

**Review by saruniitr — APPROVED** [review]
with duplicate line fixed, this looks good to me

**Review by Tllew — APPROVED** [review]
Looks good

**Pull request merged**